modules/react/common/lib/theming/getObjectProxy.ts

```diff
@@ -22,3 +26,5 @@ export function getObjectProxy<T extends {}>(target: unknown, fallback: T): T {
     },
   }) as T;
 }
+
+getObjectProxy({}, defaultCanvasTheme); //?
```

Comment:
I need explanation why this is here now.

Reply:
Accidental commit. I was running an experiment on this object.

Reply:
I'm removing.